### PR TITLE
Fix extension downloading when "Allow access to all URLs" is enabled (#26)

### PR DESCRIPTION
**Pull request by ngyikp**
#26

### CODE REVIEW

**Comment by Rob--W** [comment]
Thanks for your report, analysis and patch! Could you add the same option to the web target, right after this line: https://github.com/Rob--W/crxviewer/blob/be8fca570d1fef55bd8d1cc6521525a9c050ab55/make.js#L141 ?

**Comment by ngyikp** [comment]
Done.

File: make.js [hunk 1]
```diff
@@ -140,7 +141,8 @@ target.web = function() {
         ],
         preprocess: [
             [SRC_DIR + 'crxviewer.html', dest_dir],
-            [SRC_DIR + 'crxviewer.js', dest_dir]
+            [SRC_DIR + 'crxviewer.js', dest_dir],
+            [SRC_DIR + 'lib/crx-to-zip.js', dest_dir + 'lib/crx-to-zip.js']
```
Comment (Rob--W) [inline make.js]: Please add a comment after this line, and remove `/crx-to-zip.js` at the right (also above).

After that we're good to go.

Comment (ngyikp) [inline make.js]: Done, please take another look 👍

File: make.js [hunk 2]
```diff
@@ -140,7 +141,8 @@ target.web = function() {
         ],
         preprocess: [
             [SRC_DIR + 'crxviewer.html', dest_dir],
-            [SRC_DIR + 'crxviewer.js', dest_dir]
+            [SRC_DIR + 'crxviewer.js', dest_dir],
+            [SRC_DIR + 'lib/crx-to-zip.js', dest_dir + 'lib'] // See https://github.com/Rob--W/crxviewer/issues/26
```
Comment (Rob--W) [inline make.js]: The comma is still missing, and "// See http.." should be removed. Once merged, the repository will include references to your pull request and the commit, so there is no need to include the URL.

I suggest to squash all commits and give it a description that includes the original bug report, like this:

```
git reset master
git commit -v -m "Add crx-to-zip.js to preprocessor #26"
```

Comment (ngyikp) [inline make.js]: Sorry for the delay, please take another look

**Pull request merged**